.github/workflows/CI.yml

```diff
@@ -247,33 +247,23 @@ jobs:
 
     env:
       TEST_SUITES: testinstall
+      # CHERE_INVOKING=1 lets us start a 'login shell' (to set paths) without changing directory
+      CHERE_INVOKING: 1
 
-    steps:
-      # This sets git to use UNIX line endings. While GAP should allow Windows
-      # line endings in GAP files, we currently can't build GAP if the whole
-      # source has Windows line endings
-      - name: "Set git to use UNIX-style line endings"
-        run: |
-          git config --global core.autocrlf false
-          git config --global core.eol lf
 
+    steps:
       - uses: actions/checkout@v2
 
       - uses: gap-actions/setup-cygwin-for-gap@v1
 
       # CHERE_INVOKING=1 lets us start a 'login shell' (to set paths) without changing directory
```

Comment:
Woops, yes, this was a mistake when merging -- I moved the CHERE_INVOKING into the env just because it (hopefully) gets us closer to not having to special case cygwin